Training24/Header.h

```diff
@@ -1,17 +1,30 @@
-// ------------------------------------------------------------------------------------
+// --------------------------------------------------------------------------------------
 // Training ~ A training program for new joiners at Metamation, Batch - July 2024.
 // Copyright (c) Metamation India.
-// -------------------------------------
-//  Dhaarani Mohanasundaram
-// -------------------------------------
-// Standard Header file in Main
-// -------------------------------------
+// Dhaarani Mohanasundaram
+// --------------------------------------------------------------------------------------
 // Header.h
-// Program on main branch.
-// ------------------------------------------------------------------------------------
+// Program on branch A4.
+// Palindrome Checker: Program that checks whether the given input(Phrase and integer)
+// is a palindrome and test few cases automatically.
+// --------------------------------------------------------------------------------------
 #pragma once
+#pragma warning (disable:4996)
+
 #define HEADER_H
+#define _CRT_SECURE_NO_WARNINGS
+
+/// <summary>Function to check if a phrase is a palindrome</summary>
```

Comment:
Comments must explain only the functionality of the functions. Need not write 'Function to ...' in every comment. Make this change wherever applicable.